powertools-metrics/src/test/java/software/amazon/lambda/powertools/metrics/internal/LambdaMetricsAspectTest.java

```diff
@@ -264,7 +263,7 @@ public void exceptionWhenNoMetricsEmitted() {
         try (MockedStatic<SystemWrapper> mocked = mockStatic(SystemWrapper.class)) {
             mocked.when(() -> SystemWrapper.getenv("AWS_EMF_ENVIRONMENT")).thenReturn("Lambda");
 
-            MetricsUtils.defaultDimensionSet(new DimensionSet());
+            MetricsUtils.defaultDimensions((DimensionSet) null);
```

Comment:
nit: I see other statement not having to use the type casting, is it really needed here?

Reply:
No, its not . Just a miss